OData/src/System.Web.OData/OData/Query/Expressions/FilterBinder.cs

```diff
@@ -233,6 +237,57 @@ private Expression Bind(QueryNode node)
             }
         }
 
+        private Expression BindOpenPropertyAccessQueryNode(SingleValueOpenPropertyAccessNode openNode)
+        {
+            var prop = GetOpenTypeProperty(openNode);
+            var propertyAccessExpression = BindPropertyAccessExpression(openNode, prop);
```

Comment:
there is a same function defined as MakeFunctionCall(..), please consider to re-use the existing function.